packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -381,8 +381,8 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
           <span
             id={this._optionId}
             className={this._classNames.title}
-            aria-live="polite"
-            aria-atomic={true}
+            aria-live={hasFocus ? 'polite' : undefined}
```

Comment:
All the snapshots no longer have any aria-live or aria-atomic attributes.  Do you want to add a unit test to verify they are applied when the hasFocus is set?